src/gallia/log.py

```diff
@@ -299,7 +303,7 @@ def add_stderr_log_handler(
 
 def add_zst_log_handler(
     logger_name: str, filepath: Path, file_log_level: Loglevel
-) -> None:
+) -> logging.Handler:
```

Comment:
A major rework of the logging system is required with https://github.com/Fraunhofer-AISEC/gallia/discussions/479. This might eventually drop your proposed changes in the long term. If your suggested change is crucial for your work we could cherry pick them into a new pull request and merge them separetely.

Reply:
I strongly disagree.

First, how does #479 affect "the logging system" at all?
Second, these are just passive changes that can easily be adjusted if needed.

Reply:
As I mentioned, if you need those changes right now, please cherry pick them into a new pull request and I will merge them ASAP.

Reply:
I don't need them right now and can wait for the entire MR to be accepted, assuming this will also just be a few weeks max... :wink: